app/experimenter/normandy/tests/test_tasks.py

```diff
@@ -475,6 +475,43 @@ def test_firefox_version_no_updates_when_no_change(self):
 
         self.assertFalse(experiment.changes.filter(message="Added Version(s)").exists())
 
+    def test_live_experiment_rounds_population_percent_correctly(self):
```

Comment:
I think we should have a test that replicates the case where unnecessary duplicates are created and asserts they aren't?

Reply:
https://github.com/mozilla/experimenter/pull/3888/files#diff-2a8ff87f0439ee59f7dd865e60a59a2e0d52045a48fbcf807b7e4d532e0865f3R511
?

Reply:
Oh oh! lol I'm dumb I thought that was an asserTrue yes good yep 👍